docs/acr42/01_introduction_and_goals.md

```diff
@@ -0,0 +1,68 @@
+|                        | Scalability           | How many sequences do we expect?                                                                                                      | Q:P4 |
+|                        |                       |                                                                                                                                       |      |
+| Maintainability        | Modularity            | Wie einfach sollen einzelne Komponenten (Web access, database, config) ausgetauscht werden können?                                    | Q:M1 |
+|                        | Reusability           | Welche Komponenten sollen auch getrennt benutzt werden können (database?)                                                             | Q:M2 |
+|                        | Analysability         | Wie effektiv und effizient soll das fertige Produkt analysiert werden können?                                                         | Q:M3 |
+|                        | Modifiability         | Welche Komponten sollen angepasst werden können? The (whole?) software should be easy to modify to cope with different virus species. | Q:M4 |
+|                        | Testability           | All single components should be tested. The whole system is tested from starting an instance to the user query.                       | Q:M5 |
+
+[//]: # (| Cultural and Regional? |)
+
+[//]: # (| Legal?)
+
+[//]: # (| Security?)
+
+[//]: # ()
+
+[//]: # (Scenarios:)
+
+## Stakeholders
+
+| Role                              | Expectations                                                           |
+|-----------------------------------|------------------------------------------------------------------------|
+| Database researcher (Alex)        | ?                                                                      |                                                                      
+| Alignment researcher (Cornelius)  | ?                                                                      |                                                                      
+| Sequence researcher (Chaoran)     | ?                                                                      |                                                                    
+| Financier (Tanja, DFG?)           | ?                                                                      |                                                                   
+| Maintainer                        | Can fix bugs and add new features to software.                         |                         
+| End-user                          | Can write queries to LAPIS and get a fast result.                      |                      
+| Research facility                 | Can host the software on their own servers for their own virus specie. | 
```

Comment:
```suggestion
| Developer                        | Can fix bugs and add new features to software.                         |                         
| End-user                          | Can write queries to LAPIS and get a fast result.                      |                      
| Maintainer                        | A research facility that can host the software on their own servers for their own virus specie. | 
```
In my understanding, the "maintainer" is the one who runs the software in the end. At least that's what called it and also refer to it as in later chapters. But I'm open for a discussion on how to actually call them, since this also seems to be an important distinction.